bylaws/2-cf-membership.md

```diff
@@ -6,12 +6,20 @@ weight: 2
 
 The Commonhaus Foundation (CF) is a membership organization. The following section outlines the different types of membership and the process for becoming a member.
 
+- [Rights and requirements](#rights-and-requirements)
 - [Project Leaders and Code Owners](#project-leaders-and-code-owners)
 - [General Members](#general-members)
 - [Membership Dues and Renewal](#membership-dues-and-renewal)
 - [Inactive or Retired Members](#inactive-or-retired-members)
 - [Suspension and Revocation of Membership](#suspension-and-revocation-of-membership)
 
+## Rights and requirements
+
+Commonhaus Foundation members are natural persons, and join as individuals.
+This excludes in particular legal entities, which are exclusively represented as part of the [advisory board][].
+
+Each member has a voice, can raise issues (for all members, for all project leads, for the council), and is entitled to a vote in member decisions (which includes both elections and referendums).
```

Comment:
Non members can raise issues too I reckon and thus not a special trait given to members is it?

Reply:
Do you have any thoughts on the types of issues a non-member would need to raise with the Foundation itself as opposed to a particular project?

Reply:
I can't think of one specifically but also can't think of reason why it should be not allowed. Is the plan to limit raisih issues to the foundation to members only ?

Reply:
I can update the wording in a new PR, but this is scoped to the foundation operation itself (not individual projects). Discussions can be raised by anyone (there won't be a limit), but non-members won't be able to schedule those for a vote (for example), where a member would.